.github/workflows/publish.yml

```diff
@@ -0,0 +1,47 @@
+name: Publishing
+
+on:
+  push:
+    branches: [master]
+    tags: ['*']
+
+jobs:
+  publish-jar-tag:
+    if: startsWith(github.ref, 'refs/tags/')
```

Comment:
We can just remove it together `branches: [master]` above.

Reply:
Fixed, thanks!